Session-6/DeckOfCards/CardsDeck.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DeckOfCards
+{
+    public abstract class CardsDeck<T>
+    {
+        public List<T> cardsDeck;
+        public CardsDeck()
+        {
+            cardsDeck = new List<T>();
+        }
+        public abstract void CreateDeck();
```

Comment:
This shouldn't be public. When you create new deck, automatically it should have cards.